packages/aft/lib/src/commands/integration_tests_command.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:interact/interact.dart';
+
+import '../devices.dart';
+import '../utils/start_flutter_process.dart';
+
+/// Command to run integration tests.
+class IntegrationTestCommand extends AmplifyCommand {
+  IntegrationTestCommand(this.args);
+
+  List<String> args;
+
+  @override
+  String get description =>
+      'Runs the appropriate integration test command for the given platform';
+
+  @override
+  String get name => 'integ';
+
+  @override
+  Future<void> run() async {
+    final packageSelections = [
+      'amplify_authenticator',
+      'auth/amplify_auth_cognito'
+    ];
+    final packagePrompt = Select(
+      prompt: 'Which tests do you want to run?',
+      options: packageSelections,
+    );
+
+    final platformSelections = [
+      'VM',
+      'Web',
+    ];
+    final platformPrompt = Select(
+      prompt: 'Are you running on a VM or Web?',
+      options: platformSelections,
+    );
+
+    final deviceSelections = <String>[];
+    final devicePrompt = Select(
+      prompt: 'Select a device:',
+      options: deviceSelections,
+    );
+
+    final selectedPackage = packageSelections[packagePrompt.interact()];
+    final selectedPlatform = platformSelections[platformPrompt.interact()];
+
+    switch (selectedPlatform) {
+      case 'VM':
+        var devices = <ActiveDevice>[];
+        await startFlutterProcess(
+          [
+            'devices',
+            '--machine',
+          ],
+          selectedPackage,
+          printStream: false,
+        ).then((result) {
+          final list = json.decode(result) as Iterable;
+          devices = List<ActiveDevice>.from(
+            list.map(
+              (model) => ActiveDevice(
+                name: model['name'] as String,
+                id: model['id'] as String,
+              ),
+            ),
+          )..forEach((ActiveDevice el) {
+              if (el.name != 'Chrome') {
+                deviceSelections.add(el.name);
+              }
+            });
+        });
+
+        final devicePromptSelection = deviceSelections[devicePrompt.interact()];
+        final selectedDevice = devices
+            .firstWhere((device) => device.name == devicePromptSelection);
+
+        final results = await _executeTests(
+          [
+            'test',
+            'integration_test',
+            '-d',
+            selectedDevice.id,
+          ],
+          selectedPackage,
+          1,
+        );
+        stdout.write(results);
+        break;
+
+      // TODO(dnnoyes) get a parse driver results properly
+      case 'Web':
+        Process? process;
+        try {
+          process = await Process.start('chromedriver', ['--port=4444']);
+          stdout.write('Started chromedriver \n');
+        } on Exception catch (e) {
+          stderr
+            ..write('Tried and failed to start chromedriver.')
+            ..write(
+                'Make sure you are running chromedriver on port 4444 or that chromedriver is in your path.')
```

Comment:
```suggestion
                'Make sure you are running chromedriver on port 4444 or that chromedriver is in your PATH.')
```

Reply:
I'm not sure I understand this error message. If we fail to start chromedriver on port 4444, telling the user that they should ensure it's running doesn't make any sense in my mind.